src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButton.tsx

```diff
@@ -78,17 +77,17 @@ export const ArtworkActionsSaveButton: FC<ArtworkActionsSaveButtonProps> = ({
   if (isOpenOrUpcomingSale) {
     return (
       <ArtworkActionsWatchLotButtonFragmentContainer
-        isSaved={!!artwork.isSaved}
+        isSaved={artwork.isSavedToAnyList}
         artwork={artwork}
         onClick={handleSaveArtworkInAuction}
-        canShowRegistrationPopover={!artwork.isSavedToList}
+        canShowRegistrationPopover
```

Comment:
This is the one exception mentioned in the PR description - it wasn't clear to me why we might suppress a bidder registration popover when an artwork _is_ saved to a custom list (potentially in _addition_ to being saved in the default collection). So just rolled with `true`.

The popover itself only shows up if you started out clicking the save button while logged-out: https://github.com/artsy/force/blob/061f45bcb93fd9fee41d1ec2594798928cb397fa/src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsWatchLotButton.tsx#L65

Reply:
Yes - by 'save' i mean 'watch lot' in the context of this auction thing - whoops!